catalog/spatial/csw/spatial-csw-source-common/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,26 @@
+<!--
```

Comment:
Good idea! We should move this to a common test library that anybody can use in their Maven modules to suppress logging during unit tests. Talk to me or @oconnormi to know more :-)